### DISCUSSION TITLE
Welcome to 23-Homework4G1 Discussions!

### DISCUSSION THREAD

**Discussion by reshnashrestha** [ubsuny/23-Homework4G1#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by reshnashrestha** [comment]
@Mosaddeq107 We can discuss our homework in this discussion section. Those topics that do not need to be an issue can be discussed here.

**Reply by Mosaddeq107** [reply]
Thank you. It's a good idea.
